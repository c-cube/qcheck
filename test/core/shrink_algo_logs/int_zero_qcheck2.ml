open QCheck2

let ints_are_0 =
  Test.make ~name:"ints are 0"
    Gen.int (Log.shrinks_int @@ fun i -> i = 0)
;;
try Test.check_exn ints_are_0 with Test.Test_fail _ -> ()
