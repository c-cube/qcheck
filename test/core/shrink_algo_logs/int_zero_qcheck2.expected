fails 2724675603984413065
holds 0
fails 1362337801992206532
holds 0
fails 681168900996103266
holds 0
fails 340584450498051633
holds 0
fails 170292225249025816
holds 0
fails 85146112624512908
holds 0
fails 42573056312256454
holds 0
fails 21286528156128227
holds 0
fails 10643264078064113
holds 0
fails 5321632039032056
holds 0
fails 2660816019516028
holds 0
fails 1330408009758014
holds 0
fails 665204004879007
holds 0
fails 332602002439503
holds 0
fails 166301001219751
holds 0
fails 83150500609875
holds 0
fails 41575250304937
holds 0
fails 20787625152468
holds 0
fails 10393812576234
holds 0
fails 5196906288117
holds 0
fails 2598453144058
holds 0
fails 1299226572029
holds 0
fails 649613286014
holds 0
fails 324806643007
holds 0
fails 162403321503
holds 0
fails 81201660751
holds 0
fails 40600830375
holds 0
fails 20300415187
holds 0
fails 10150207593
holds 0
fails 5075103796
holds 0
fails 2537551898
holds 0
fails 1268775949
holds 0
fails 634387974
holds 0
fails 317193987
holds 0
fails 158596993
holds 0
fails 79298496
holds 0
fails 39649248
holds 0
fails 19824624
holds 0
fails 9912312
holds 0
fails 4956156
holds 0
fails 2478078
holds 0
fails 1239039
holds 0
fails 619519
holds 0
fails 309759
holds 0
fails 154879
holds 0
fails 77439
holds 0
fails 38719
holds 0
fails 19359
holds 0
fails 9679
holds 0
fails 4839
holds 0
fails 2419
holds 0
fails 1209
holds 0
fails 604
holds 0
fails 302
holds 0
fails 151
holds 0
fails 75
holds 0
fails 37
holds 0
fails 18
holds 0
fails 9
holds 0
fails 4
holds 0
fails 2
holds 0
fails 1
holds 0
